frontend/src/modules/APIVersion/utils.ts

```diff
@@ -0,0 +1,16 @@
+import { APIVersion } from './interface';
+
+export const isUpperOrEqualCurrentAPIVersion = (
+  version: string,
+  currentAPIVersion?: APIVersion,
+) => {
+  if (currentAPIVersion === undefined) {
+    return null;
+  }
+  return (
+    version.localeCompare(currentAPIVersion?.version.replace(/\D*$/, ''), undefined, {
+      numeric: true,
+      sensitivity: 'base',
+    }) < 0
```

Comment:
As you already have [`semver`](https://www.npmjs.com/package/semver) as subdependency, it would probably be better to use it over comparing strings.

Reply:
Yes, thank you, I hadn't noticed that. Indeed, I also think it's better to use it. With this modification, I've added a few unit tests. :+1: